test/ex_double_entry_stress_test.exs

```diff
@@ -1,12 +1,14 @@
 defmodule ExDoubleEntryStressTest do
-  use ExDoubleEntry.DataCase
+  use ExDoubleEntry.DataCase, async: false
   import ExDoubleEntry
   import Ecto.Query
   alias ExDoubleEntry.{Line, MoneyProxy}
 
-  @processes 5
+  @moduletag :stress_test
```

Comment:
There is still something wrong with this test. It fails randomly on deadlocks 🤔

Reply:
...or with the code because I see deadlocks when stress testing via client app when using multiple `iex` consoles. Still investigating.